include/ignition/utils/config.hh.in

```diff
@@ -15,3 +15,10 @@
 #cmakedefine IGNITION_UTILS_BUILD_TYPE_PROFILE 1
 #cmakedefine IGNITION_UTILS_BUILD_TYPE_DEBUG 1
 #cmakedefine IGNITION_UTILS_BUILD_TYPE_RELEASE 1
+
```

Comment:
should we add header guards to this file?

Reply:
Good call, added in [5fee873](https://github.com/ignitionrobotics/ign-utils/pull/50/commits/5fee87349cdfa442df89e87c9c13a34628c5c95b)